### PR TITLE
[Molecule][idrac_storage_controller] - Test case automation

### PR DESCRIPTION
**Pull request by singh-lovepreet1**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — DISMISSED** [review]

File: roles/idrac_storage_controller/molecule/__fetch_volumes.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+- name: Pre_req - View all volumes
+  dellemc.openmanage.idrac_storage_volume:
+    idrac_ip: "{{ lookup('env', 'IDRAC_IP') }}"
+    idrac_user: "{{ lookup('env', 'IDRAC_USER') }}"
+    idrac_password: "{{ lookup('env', 'IDRAC_PASSWORD') }}"
+    validate_certs: false
+    state: "view"
+  register: result
```
Comment (sachin-apa) [inline roles/idrac_storage_controller/molecule/__fetch_volumes.yml]: Can we register the var on the format role_name_var, PLease update other files as required.

File: roles/idrac_storage_controller/molecule/__get_job_details.yml
Comment (sachin-apa) [inline roles/idrac_storage_controller/molecule/__get_job_details.yml:1]: Please fix jina3 template warning un the file.

Comment (singh-lovepreet1) [reply roles/idrac_storage_controller/molecule/__get_job_details.yml:1]: Resolved

File: roles/idrac_storage_controller/molecule/__fetch_volumes.yml (hunk 1 above)
Comment (singh-lovepreet1) [reply roles/idrac_storage_controller/molecule/__fetch_volumes.yml]: Updated

**Review by sachin-apa — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**